lib/server-builder.js

```diff
@@ -32,13 +32,17 @@ class ServerBuilder {
     const buildConfiguration = args.buildConfiguration || {};
 
     const versionConfiguration = buildConfiguration.toolsVersions || {};
-    const serverVersions = {};
-    for (const key of VERSION_KEYS) {
-      if (versionConfiguration[key]) {
-        serverVersions[key] = versionConfiguration[key];
-      }
-    }
-    this.serverVersions = serverVersions;
+    this.serverVersions = Object.entries(versionConfiguration)
```

Comment:
the _reduce function for lodash lib could make it all much shorter. See https://github.com/appium/appium-idb/blob/2dce71d2b21b5d6bf95fcd5bf74e4c6bef0549ea/lib/helpers.js#L56